spec/requests/api/v1/relationships/merchants_items_spec.rb

```diff
@@ -11,7 +11,8 @@
         expect(response).to be_successful
 
         merchants = JSON.parse(response.body, symbolize_names: true)
+        binding.pry
```

Comment:
Be very careful with committing `binding.pry`. This is in a test and not likely to have any impact in the production code so while it still needs to be removed it doesn't present an imminent problem with this PR. 

Let's just say someone who is very close to me and shares my name and is me once committed a pry much further than it should've been in the production pipeline ;-)